apps/web/components/v2/apps/App.tsx

```diff
@@ -0,0 +1,337 @@
+      <div className="relative flex-1 flex-col items-start justify-start px-4 md:flex md:px-8 lg:flex-row lg:px-0">
+        {hasImages && (
+          <div className="flex-2 mb-4 -ml-4 -mr-4 flex w-auto snap-x snap-mandatory flex-row overflow-auto whitespace-nowrap bg-gray-100  p-4 md:mb-8 md:-ml-8 md:-mr-8 md:p-8 lg:mx-0 lg:mb-0 lg:max-w-2xl lg:flex-col lg:rounded-md">
+            {images &&
+              images.map((img) => (
+                <img
+                  key={img}
+                  src={img}
+                  alt={`Screenshot of app ${name}`}
+                  className="mr-4 max-h-80 max-w-[90%] snap-center rounded-md last:mb-0 lg:mb-4 lg:mr-0  lg:max-w-full"
+                />
+              ))}
+          </div>
+        )}
+        <div
+          className={classNames(
+            "sticky top-0 max-w-xl flex-1 pb-12 text-sm lg:pb-0",
+            hasImages && "lg:ml-8"
+          )}>
+          <div className="mb-8 flex">
+            <header>
+              <div className="mb-4 flex items-center">
+                <span className="block rounded-sm bg-gray-100 p-2">
+                  <img className="min-h-16 min-w-16 h-16 w-16" src={logo} alt={name} />
+                </span>
+                <h1 className="font-cal ml-4 text-3xl text-gray-900">{name}</h1>
+                {isProOnly && user?.plan === "FREE" ? (
+                  <Badge className="ml-2" variant="default">
+                    PRO
+                  </Badge>
+                ) : null}
+              </div>
+              <h2 className="text-sm text-gray-600">
+                <span className="bg-gray-100 p-1 text-xs capitalize text-gray-800">{categories[0]}</span> •{" "}
+                {t("published_by", { author })}
+              </h2>
+            </header>
+          </div>
+          {!isLoading ? (
+            isGlobal || (installedAppCount > 0 && allowedMultipleInstalls) ? (
+              <div className="flex space-x-3">
+                <Button StartIcon={Icon.FiCheck} color="secondary" disabled>
+                  {installedAppCount > 0
+                    ? t("active_install", { count: installedAppCount })
+                    : t("globally_install")}
+                </Button>
+                {!isGlobal && (
+                  <InstallAppButton
+                    type={type}
+                    isProOnly={isProOnly}
+                    render={({ useDefaultComponent, ...props }) => {
+                      if (useDefaultComponent) {
+                        props = {
+                          onClick: () => {
+                            mutation.mutate({ type });
+                          },
+                          loading: mutation.isLoading,
+                        };
+                      }
+                      return (
+                        <Button
+                          StartIcon={Icon.FiPlus}
+                          {...props}
+                          // @TODO: Overriding color and size prevent us from
```

Comment:
Let me know if this is the best to do for now @sean-brydon. Currently the types of `InstallAppButton` use the old types of button for `props`. Because of this the button component gives a TS error. By manually overriding color + size, everything is fine. I was unsure whether it would make sense to only for changes the types would make sense to duplicate the `InstallAppButton`, since this also brings overhead in maintenance etc.